sodetlib/operations/bias_dets.py

```diff
@@ -201,3 +201,347 @@ def bias_to_rfrac(S, cfg, rfrac=0.5, bias_groups=None, iva=None,
 
     return biases
 
+def biasstep_rebias(
+        S, cfg, target_percentage_rn = 0.5, bias_groups=None,
+        math_only=False):
```

Comment:
Plotting should be optional, and plots should be published.

Reply:
fixed